index.html

```diff
@@ -296,6 +317,31 @@
             document.getElementById("euler").value);
         q.setFromEuler(e);
     }
+    else if (inputMode ==5)
+    {
+        var m = new THREE.Matrix4();
+        var p1x = document.getElementById("p1x").value;
```

Comment:
Feel free to introduce a shorter helper function for getElementById...value

Reply:
It was added with getVector().